bundles/org.openhab.binding.gpio/README.md

```diff
@@ -22,12 +22,25 @@ sudo raspi-config
 
 -> Interfacing Options --> Remote GPIO --> YES --> OK --> Finish
 
+Note: if you are setting this up on a Raspberry Pi running openHabian, raspi-config is no longer available. However, you can still create the service config file manually:
+
+```
+sudo mkdir -p /etc/systemd/system/pigpiod.service.d/
+sudo nano /etc/systemd/system/pigpiod.service.d/public.conf
+```
+      [Service]
+      ExecStart=
+      ExecStart=/usr/bin/pigpiod
```

Comment:
Why are there two `ExecStart`s?

Reply:
Because this is what the raspi-config script adds to the pigpiod config file and I copied verbatim (in case it matters).
https://github.com/RPi-Distro/raspi-config/blob/74885481d8f311ab3875b6601153132552b41d67/raspi-config#L1616

Reply:
It seems an error to me. Can you test if it still works if you remove the empty `ExecStart=`?

Reply:
It seems to be to override the default values. See https://bugzilla.redhat.com/show_bug.cgi?id=756787. So I guess it is to be sure no default settings are present. That would mean even if @SebBoulet would test this on his system and it works, it might not work for other people who have other configurations. So I would say this is legitimate use case.

Reply:
I didn't find that info when doing a quick search. Thanks!